src/main/java/ui/Printer.java

```diff
@@ -300,4 +302,75 @@ public void printColors() {
         print(ConsoleColorsEnum.BRIGHT_MAGENTA_TEXT + "TEST\n" + reset);
         print(ConsoleColorsEnum.BOLD_MAGENTA_TEXT + "TEST\n" + reset);
     }
+
+    public void printStartHelpMessage() {
+        print(NEWLINE + "Here are some basics commands to get you started: "
+            + NEWLINE + "\t\"/add\": Add your own cheat sheet"
+            + NEWLINE + "\t\"/list\": List all cheat sheet you currently own, including those we preloaded for you"
+            + NEWLINE + "\t\"/view\": View the contents of a specific cheat sheet, we recommend you to use the list "
+            + "command before this. " + NEWLINE + "\t\t\t You can view the contents using cheat sheet index or name."
+            + NEWLINE + "Enter \"/help\" to see the details of all commands");
+    }
+
+    private void printRepeatedHelpMessagePartOne(String commandType) {
+        print("The " + commandType + " does not work alone."
+            + NEWLINE + "We need some necessary information from you before we could add your cheat sheet."
+            + NEWLINE + "Don't worry, you do not have to enter the command again."
+            + NEWLINE + "We would prompt you to enter the necessary information, one by one.");
+    }
+
+    private void printRepeatedHelpMessagePartTwo() {
+        print("You also could enter the full command next time if you do not wish to be prompted to enter fields "
+            + "one by one" + NEWLINE + "\"/help\" to see more details on the structure of full command."
+            + NEWLINE + "To stop messages like this from appearing again, input this command: \"/set /m remove\"");
+    }
+
+    public void printAddCommandHelpMessage() {
+        printRepeatedHelpMessagePartOne("ADD");
+        print("NAME is necessary while SUBJECT isn't. <<enter>> to skip field."
+            + NEWLINE + "We will automatically assign \"Unsorted\" as subject if no subject is provided." + NEWLINE);
+        printRepeatedHelpMessagePartTwo();
+    }
+

```

Comment:
Perhaps you could seperate each addition on a separate line for easier readability since there quite a few of them?

Reply:
Oh yea, thanks for the suggestion